server/sonar-web/src/main/js/helpers/handlebars/componentDashboardPermalink.js

```diff
@@ -35,5 +35,5 @@ module.exports = function (componentKey, dashboardKey) {
   const query = params.map(function (p) {
     return p.key + '=' + encodeURIComponent(p.value);
   }).join('&');
-  return '/dashboard/index?' + query;
+  return window.baseUrl + '/dashboard/index?' + query;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Convert this concatenation to the use of a template. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=javascript%3AS3512)